Alicerce/Sources/Extensions/UICollectionView.swift

```diff
@@ -22,4 +22,8 @@ public extension UICollectionView {
     public func register<T: UICollectionViewCell>(_ cellType: T.Type) where T: ViewCellProtocol {
         register(cellType, forCellWithReuseIdentifier: cellType.reuseIdentifier)
     }
+    
+    public func register<T: AnyObject>(_ viewType: T.Type, forSupplementaryViewOfKind kind: String) where T: ViewCellProtocol {
+        register(viewType.self, forSupplementaryViewOfKind: kind, withReuseIdentifier: viewType.reuseIdentifier)
```

Comment:
You're right. Not sure why I added that in 😕 
I've removed in 3b1d945bb496e940352f908ef2ee9655dfaed540.